config/jobs/kubernetes/sig-testing/periodic-kubernetes-compatibility-version-e2e.yaml

```diff
@@ -0,0 +1,51 @@
+periodics:
+- interval: 24h
+  cluster: k8s-infra-prow-build
+  name: pull-kubernetes-e2e-kind-emulated-version
+  annotations:
+    testgrid-dashboards: sig-testing-kind
+    testgrid-tab-name: compatibility-version-test
+    description: Uses kubetest & kind to run e2e tests from older (n-1..3) kubernetes releases against a latest kubernetes master components w/ --emulated-version=<n-1..3> set.
+    testgrid-alert-email: aprindle@google.com
```

Comment:
can we route to a rotation? (maybe ok for now, add a TODO?)

Reply:
Good point, added a TODO comment for this now

Reply:
In some places we have a SIG alerts list, I don't know if api machinery or architecture have one

it can route to multiple emails anyhow, so if you're personally working on this and want to be alerted to start that's common and fine

Reply:
I am personally working on this and would like to be alerted to start so I think just having `aprindle@google.com` there for now should be ok in that case.